phone/src/apps/match/hooks/useMatchActions.ts

```diff
@@ -0,0 +1,61 @@
+import { FormattedProfile, MatchEvents, Profile } from '../../../../../typings/match';
+import dayjs from 'dayjs';
+import { useTranslation } from 'react-i18next';
+import { fetchNui } from '../../../utils/fetchNui';
+import { ServerPromiseResp } from '../../../../../typings/common';
+import { useSetFormattedProfiles } from './state';
+import { useSnackbar } from '../../../ui/hooks/useSnackbar';
+import { useCallback } from 'react';
+
+export const useMatchActions = () => {
+  const { t } = useTranslation();
+  const setProfiles = useSetFormattedProfiles();
+  const { addAlert } = useSnackbar();
+
+  const formatProfile = (profile: Profile): FormattedProfile | null => {
+    return {
+      ...profile,
+      tagList: profile.tags.split(',').filter((t) => t), // remove any empty tags
+      lastActiveFormatted: dayjs.unix(profile.lastActive).format(t('DATE_TIME_FORMAT')),
+      viewed: false,
+    };
+  };
```

Comment:
This is a pure function that doesn't need state context, I would have it outside of the hook. Also is there any reason we have null as a possible return type?

Reply:
This is probably not even used at all. Removing it.